internal/api/core/describe.go

```diff
@@ -449,6 +475,15 @@ func describeStruct(customType reflect.Type, knownPackages map[string]StructDesc
 	return description, nil
 }
 
+func isFieldRequired(field reflect.StructField) bool {
+	_, ok := field.Tag.Lookup("required")
```

Comment:
We want to discourage tags like `required:"false"`,
so you should probably ensure that the tag value is empty.